test/spec/convertToHTML.js

```diff
@@ -177,118 +177,6 @@ describe('convertToHTML', () => {
     expect(result).toBe('<ul><li>top level</li><ul><li>nested one level</li><ul><li>nested two levels</li></ul></ul><li>back to top level</li></ul>');
   });
 
-  it('escapes HTML in text of blocks', () => {
```

Comment:
hmm, what's the context on removing these? this is a pretty important XSS protection